.github/PULL_REQUEST_TEMPLATE/release_pr_template.md

```diff
@@ -0,0 +1,12 @@
+# Changes
+
+Link here all the PRs that are included in this release
+
+# Release-candidates
+
+Link here the release-candidates that were deployed as part of this release
+
+# Checklist
+
+- [ ] I've read and followed the release process described in https://github.com/HathorNetwork/ops-tools/blob/master/docs/release-guides/wallet-service.md#stable-release
```

Comment:
Link is from wallet service release process

Reply:
Removed in 50213095c637848b9edcd20ec49af774453aad4d